wallet/src/Dscp/Wallet/Backend.hs

```diff
@@ -22,12 +22,14 @@ createWalletFace ::
 createWalletFace serverAddress sendEvent = buildComponent_ "Wallet" $ do
     sendStateUpdateEvent sendEvent
     wc <- createWitnessClient serverAddress
+    (dummySK, _) <- keyGen
```

Comment:
I can imagine the day when estimated fees would vary across restarts depending on the moon phase.
(I do not insist, but we have `detGen` for deterministic generation)